test cases/windows/13 resources with custom targets/prog.c

```diff
@@ -0,0 +1,14 @@
+#include<windows.h>
+
+#define MY_ICON 1
+
+int APIENTRY
+WinMain(
+    HINSTANCE hInstance,
+    HINSTANCE hPrevInstance,
+    LPSTR lpszCmdLine,
+    int nCmdShow) {
+    HICON hIcon;
+    hIcon = LoadIcon(NULL, IDI_APPLICATION);
```

Comment:
This test (and the one it's copied from) doesn't seem to be testing what it claims to test. I think this line should be:

```
    hIcon = LoadIcon(GetModuleHandle(NULL), MAKEINTRESOURCE(MY_ICON));
```

Reply:
I created PR #2851 to fix this in the original test case

Reply:
@jon-turney does that also apply to the original test (`5 resources`) too, then? I'm not very well versed in the win32 API; just copied this part verbatim and didn't think much of it.